frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -46,258 +28,39 @@ const StyledButton = styled(Button)`
-`
-
 export default function ViewFacility() {
   const dispatch = useDispatch()
   const history = useHistory()
   const params = useParams<{ facilityID: string }>()
-  const {
-    ViewStartDate,
-    ViewEndDate,
-    createSuccess,
-    createFailure,
-    isLoading,
-    facilityBookings,
-    selectedFacilityName,
-    selectedFacilityId,
-  } = useSelector((state: RootState) => state.facilityBooking)
+  const { selectedFacilityName, selectedFacilityId } = useSelector((state: RootState) => state.facilityBooking)
 
   useEffect(() => {
     dispatch(SetIsLoading(true))
     dispatch(fetchFacilityNameFromID(parseInt(params.facilityID)))
-    dispatch(getAllBookingsForFacility(ViewStartDate, ViewEndDate, parseInt(params.facilityID)))
     if (selectedFacilityId == 0) {
       dispatch(setSelectedFacility(parseInt(params.facilityID)))
     }
-    return () => {
-      dispatch(setViewDates(new Date(), parseInt(params.facilityID)))
-    }
   }, [])
 
-  const fetchTelegram = async (booking) => {
-    try {
-      fetch(DOMAIN_URL.FACILITY + ENDPOINTS.TELEGRAM_HANDLE + '/' + booking.userID, {
-        method: 'GET',
-        mode: 'cors',
-      })
-        .then((resp) => resp.json())
-        .then((data) => {
-          if (data.telegramHandle === '' || data.telegramHandle === undefined) {
-            console.log(data.err)
-          } else {
-            openTelegram(data.telegramHandle)
-          }
-        })
-    } catch (err) {
-      console.log(err)
-    }
-  }
-
-  const openTelegram = (userID) => {
-    const site = 'https://telegram.me/' + userID
-    window.open(site)
-  }
-
-  const MyBookingIcon = (
-    <img
-      src={bookingsIcon}
+  const MyBookingButton = (
+    <Button
+      state="primary"
+      text="Bookings"
+      type="button"
+      size="small"
       onClick={() => {
         history.push(PATHS.VIEW_MY_BOOKINGS_USERID + '/' + localStorage.getItem('userID'))
       }}
     />
   )
 
-  const getHumanReadableTime = (eventStartTime: number) => {
-    const date = new Date(eventStartTime * 1000)
-    let hour = date.getHours().toString()
-    if (hour.length == 1) {
-      hour = '0' + hour
-    }
-    let minutes = date.getMinutes().toString()
-    if (minutes.length == 1) {
-      minutes = '0' + minutes
-    }
-
-    return hour + minutes
-  }
-
-  const getHumanReadableDate = (eventTime: number) => {
-    const date = new Date((eventTime + 28800) * 1000)
-    const day = date.getUTCDate()
-    const monthInt = date.getUTCMonth() + 1
-
-    return '[' + day + '/' + monthInt + ']'
-  }
-
-  const AlertSection = (
-    <AlertGroup>
-      {createSuccess && createFailure && (
-        <Alert message="Successful" description="Yay yippe doodles" type="success" closable showIcon />
-      )}
-      {createFailure && !createSuccess && <Alert message="Not Successful Boohoo :-(" type="error" closable showIcon />}
-    </AlertGroup>
-  )
-
   return (
     <div style={{ backgroundColor: '#fafaf4' }}>
```

Comment:
is this necessary? since global styles should already set background to fafaf4?

Reply:
Fixed in 125ce12f5